### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ashpil**
Hey, 

Here's this new request because I don't know how to use git properly and I wanted to switch to the unforked repo.

I changed a couple of stuff since my last request, mainly just cleaning up the CSS variables, adding the fonts, and changing the look of the mobile menu.

This is how desktop looks at the moment:

image

And this is mobile:

image

Not sure if this is how exactly it'll look on mobile, but I like something like that - takes up much less space than the previous version.

Also, I changed `:focus` to `:focus-visible`.

### CODE REVIEW

File: app/assets/stylesheets/base/_forms.scss [hunk 1]
```diff
@@ -12,21 +7,20 @@ fieldset {
 
 legend {
   font-weight: 600;
-  margin-bottom: var(--spacing--small);
+  margin-bottom: $spacing-small;
```
Comment (hernanat) [inline app/assets/stylesheets/base/_forms.scss:10]: 👏

**Comment by hernanat** [comment]
love the mobile look!

File: app/assets/stylesheets/base/_buttons.scss [hunk 2]
```diff
@@ -5,30 +5,31 @@ button,
...
+  margin: 0.5rem 0;
 
   &:hover {
     background-color: $action-color--alt;
   }
 
-  &:focus {
-    outline: var(--focus-outline);
-    outline-offset: var(--focus-outline-offset);
+  &:focus-visible, &:-moz-focusring {
```
Comment (hernanat) [inline app/assets/stylesheets/base/_buttons.scss:31]: awesome. would you mind attaching a screenshot of this in action to the PR?

**Comment by ashpil** [comment]
Just realized this might actually conflict with some of the CSS changes you did for the submissions stuff, @hernanat.

**Comment by hernanat** [comment]
> Just realized this might actually conflict with some of the CSS changes you did for the submissions stuff, @hernanat.

if there is anything obvious that sticks out / suggestions you have for anything in my PR I can make the changes to better reflect the direction you're taking us in :)

**Comment by hernanat** [comment]
Screen Shot 2020-05-14 at 4 22 56 PM
we appear to be getting a good number of errors related to the fonts. I'll look into it and see what's going on.

**Comment by ashpil** [comment]
> 
> 
> Screen Shot 2020-05-14 at 4 22 56 PM
> we appear to be getting a good number of errors related to the fonts. I'll look into it and see what's going on.

Odd. Worked fine on my machine.

**Comment by hernanat** [comment]
> if there is anything obvious that sticks out / suggestions you have for anything in my PR I can make the changes to better reflect the direction you're taking us in :)

more on this -- we can get your PR in before mine and if there are any conflicts they'll show up in git. For things that are completely new in my pr we can change them accordingly as well.

**Comment by hernanat** [comment]
> Odd. Worked fine on my machine.

**Comment by hernanat** [comment]
> Odd. Worked fine on my machine.

Yeah it was an issue with the fonts folder not being a part of the assets loading path. I've made the change and pushed it to your branch. All good now :)

File: app/assets/stylesheets/base/_variables.scss [hunk 3]
```diff
@@ -14,47 +16,77 @@ $action-color--alt: $blue-light;
...
+	src: url('cmunti.eot?#iefix') format('embedded-opentype'),
+		 url('cmunti.woff') format('woff'),
+		 url('cmunti.ttf') format('truetype'),
+		 url('cmunti.svg#cmunti') format('svg');
+	font-weight: normal;
+	font-style: italic;
+}
+
+
+@font-face {
```
Comment (hernanat) [inline app/assets/stylesheets/base/_variables.scss:59]: we could also use the [font-face](https://www.bourbon.io/docs/latest/#font-face) mixin that comes with bourbon which could help make this a bit more succinct

**Comment by ashpil** [comment]
image

Here's the current implementation of the keyboard only focus. `:focus-visible` seems to be sparsely supported, so I'm using a workaround that's somewhat awkward. Try it out. I think any sort of ideal solution is either a few years out or involves JavaScript. 

I also changed it to the Bourbon font-face declaration.

**Comment by ashpil** [comment]
Squashed and commited! @hernanat

**Pull request merged**